src/commonmark/en/content/user/data-visualizer.md

```diff
@@ -2,54 +2,323 @@
+## Share a visualization
+
+Sharing settings can be accessed by clicking **File** \> **Share**. Change sharing settings for the user groups you want to modify, the available settings are:
+
+- **Can edit and view**: Can view and edit the visualization.
+
+- **Can view only**: Can only view the visualization.
+
+- **No access**: Won't have access to the visualization. This
+  setting is only applicable to **Public access** and **External access**.
+
+New users can be added by searching for them by name under `Add users and user groups`.
+
+![](resources/images/data-visualizer/data-visualizer-share-dialog.png)
+
+## Download
+
+Visualizations can be downloaded using the **Download** menu. All visualization types support `Graphics` and `Plain data source` downloads, except for the `Pivot table` type that can be downloaded as `Table layout` and `Plain data source`.
+
+### `Graphics` download
+
+Downloads an image (.png) or a PDF (.pdf) file to your computer.
+
+### `Table layout` download
+
+Downloads a Excel (.xls), CSV (.csv) or HTML (.html) file to your computer.
+
+### `Plain data source` download
+
+You can download the data source behind a chart in JSON, XML, Excel,
+CSV, JXRML or Raw data SQL formats with different identification schemes
+(ID, Code, and Name). The data document uses identifiers of the
+dimension items and opens in a new browser window to display the URL of
+the request to the Web API in the address bar. This is useful for
+developers of apps and other client modules based on the DHIS2 Web API
+or for those who require a plan data source, for instance for import
+into statistical packages.
+
+<table>
+<caption>Available formats</caption>
+<colgroup>
+<col style="width: 18%" />
+<col style="width: 33%" />
+<col style="width: 47%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th><p>Format</p></th>
+<th><p>Action</p></th>
+<th><p>Description</p></th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td><p>JSON</p></td>
+<td><p>Click <strong>JSON</strong></p></td>
+<td><p>Downloads JSON format based on the <strong>ID</strong>, <strong>Code</strong> or <strong>Name</strong> property.</p>
+</tr>
+<tr class="even">
+<td><p>XML</p></td>
+<td><p>Click <strong>XML</strong></p></td>
+<td><p>Downloads XML format based on the <strong>ID</strong>, <strong>Code</strong> or <strong>Name</strong> property.</p>
+</tr>
+<tr class="odd">
+<td><p>Microsoft Excel</p></td>
+<td><p>Click <strong>Microsoft Excel</strong></p></td>
+<td><p>Downloads Microsoft Excel format based on the <strong>ID</strong>, <strong>Code</strong> or <strong>Name</strong> property.</p>
+</tr>
+<tr class="even">
+<td><p>CSV</p></td>
+<td>Click <strong>CSV</strong></td>
+<td><p>Downloads CSV format based on the <strong>ID</strong>, <strong>Code</strong> or <strong>Name</strong> property.</p>
+</tr>
+<tr class="odd">
+<td><p>XML data value set</p></td>
+<td><p>Click <strong>Advanced > XML</strong></p></td>
+<td><p>TODO: NEEDS INFO</p></td>
```

Comment:
@janhenrikoverland , would you know what this export (and the one below) does? Not sure how to explain this.